pagopa/io-app
# chore: [IOBP-606] Add new payments section

## Short description
This PR introduces the new payments section in the home bottom navigation, displayed with **New wallet section** FF enabled.

## List of changes proposed in this pull request
- Added `PaymentsHomeScreen` in the main navigator
- Added `Payments` button in the home tab bar
- Removed `Payments` playgrounds
- Temporarily removed barcode scan screen from the tab bar in order to make room for the payments section 

## How to test
From the **Profile** section, enable the **New wallet section** FF. Check that the new screens are rendered correctly.

## Preview

https://github.com/pagopa/io-app/assets/6160324/73398682-eef2-4e92-a052-6481da17d9a8
